ui/js/auth/auth-utils.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+import { WebAuth } from 'auth0-js';
+import { fromNow } from 'taskcluster-client-web';
+
+export const webAuth = new WebAuth({
+    clientID: 'q8fZZFfGEmSB2c5uSI8hOkKdDGXnlo5z',
+    domain: 'auth.mozilla.auth0.com',
+    responseType: 'id_token token',
+    audience: 'login.taskcluster.net',
+    redirectUri: `${location.protocol}//${location.host}/login.html`,
+    scope: 'full-user-credentials openid profile email'
+});
+
+export const userSessionFromAuthResult = (authResult) => {
+    const expiresAt = JSON.stringify((authResult.expiresIn * 1000) + Date.now());
+    const userSession = {
```

Comment:
Do any of these contain LDAP groups or similar, that we can at a later date pass to the backend for bug 1273092?

Or will we need to supplement this payload with what can be gotten from the taskcluster APIs instead?

Reply:
You can pass these tokens either to the Auth0 Management API or the IAM team's Person API to get that information.  That's a better fix than looking at the Taskcluster roles associated with the returned credentials.